tests/test_compartmentfitter.py

```diff
@@ -614,10 +616,10 @@ def testParallel(self, w_benchmark=False):
 
 if __name__ == '__main__':
     tcf = TestCompartmentFitter()
-    tcf.testTreeStructure()
+    # tcf.testTreeStructure()
```

Comment:
why do you comment out tests? ;)

Reply:
This is just for testing specific tests, with pytest, they all run anyway.